airbyte-integrations/connectors/source-zendesk-talk/unit_tests/test_source.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import MagicMock
+
+import pytest
+from airbyte_cdk.models import AirbyteConnectionStatus, Status
+from airbyte_cdk.sources.streams.http import HttpStream
+
+from source_zendesk_talk import SourceZendeskTalk
+
+
+@pytest.fixture
+def patch_base_class(mocker):
```

Comment:
If I'm not mistaken you are not patching anything here. I'd rather suggest declaring a new fixture for each kind of config you'd like to test.